### PR TITLE
Update Aspects to use <del>

### PR DESCRIPTION
**Pull request by agude**

### CODE REVIEW

File: _chapters/02-world-building.md [hunk 1]
```diff
@@ -28,24 +28,22 @@ space travel, both, or neither.
 
 For example, these aspects could set the stage for a sweeping fantasy epic:
 
----------- ----------------------------------------------
-    Genre: **Magic Fades at the Dawn of the Age of Men**
- Conflict: **Ancient Evil Returns After Millennia**
----------- ----------------------------------------------
+|:----------|:----------------------------------------------|
```
Comment (charles-uno) [inline _chapters/02-world-building.md:31]: Have you confirmed that this works? I thought it didn't

Comment (charles-uno) [inline _chapters/02-world-building.md:31]: Was the alignment change intentional?

```suggestion
|----------:|:----------------------------------------------|
```

File: _config.yml [hunk 2]
```diff
@@ -44,6 +44,6 @@ exclude:
     - redirects.json
 
 dice:
-  pos: <span style="color:ForestGreen">+1</span>
-  neg: <span style="color:Red">-1</span>
-  blank: <span style="color:black">_</span>
+  pos: '<img src="/images/plus.svg" alt="+1" class="die positive-die" style="height: 1em; width: 1em; vertical-align: middle;">'
```
Comment (charles-uno) [inline _config.yml:47]: I'll be back for these later with CSS and bracket macros

**Review by charles-uno — APPROVED** [review]

File: _chapters/02-world-building.md (hunk 1 above)
Comment (agude) [reply _chapters/02-world-building.md:31]: They work, they generate a table without header row.

**Comment by agude** [comment]
Created a bug to track the table issue: #33

File: _chapters/02-world-building.md (hunk 1 above)
Comment (agude) [reply _chapters/02-world-building.md]: But see #33.

**Review by charles-uno — APPROVED** [review]

**Pull request merged**